### ISSUE TITLE
unreadable data when used with a phone (badges and cards alike)

### ISSUE THREAD

**Issue by anubisg1** [Kibibit/hass-kibibit-theme#32]
See the attached screenshot. Default theme vs kibibit

![image](https://user-images.githubusercontent.com/4518931/105369994-64c19e80-5c03-11eb-9d03-7957f08c3540.png)

![image](https://user-images.githubusercontent.com/4518931/105370072-7a36c880-5c03-11eb-921f-0437bce2cc15.png)

**Comment by Thatkookooguy** [comment]
Hi @anubisg1 ! thanks for opening this!

Unfortunately, this is just because of the different font, and not an actual bug. Every font affects the smallest screen size it can compensate. I don't change the font size at all from the default hassio is setting in this theme.

If you won't add the additional CSS resource (which loads the comfortaa font), and keep the default font, it will have the same size as the default font and everything will look good in your phone resolution.

On the mobile app, the default font-size is not configurable on the app itself. But **it is affected by the system's text size**. If you go to Settings -> Display -> text size, you can change the size of the text in your phone, which will change the size of the font in the app. The problem is, that it affects everything else on your phone as well.

I added some photos as examples at the end of this reply.

Sorry but those are the best answers I can give :-/

TL;DR:
- don't add the comfortaa font to keep the regular font and it relative sizes
- change the font size on your phone to change the font size in the app

### Settings -> Display -> Text Size
![image](https://user-images.githubusercontent.com/10427304/106328380-6258e780-6288-11eb-9006-8285928ba6e8.png)

### Regular font size
![image](https://user-images.githubusercontent.com/10427304/106328344-51a87180-6288-11eb-8e92-196df98b654d.png)

### When setting my phone to XXL font size
![image](https://user-images.githubusercontent.com/10427304/106328246-26be1d80-6288-11eb-8bdf-1661868ec0fe.png)

### When setting my phone to XXS font size
![image](https://user-images.githubusercontent.com/10427304/106328299-3d647480-6288-11eb-985c-3760c9a7f858.png)

**Comment by anubisg1** [comment]
thank you for your response. the reason why i posted this here is because on iphone is rendering correctly and on a much bigger and much higher pixel density on the galaxy s9 was rendering wrong. i'll check your suggestion. thanks again!
